releases.json

```diff
@@ -759,6 +759,7 @@
   "ludocode-mpack": {
     "versions": [
       "1.0-1"
+      "1.1-1"
```

Comment:
The newest one should be at the top, and this is missing a comma between entries.

Reply:
Thanks for pointing this out - I forgot it was a json file & not yaml so it "looked" right.  The error message in the tools/sanity_checks.py when I ran it locally looked like it was a python issue vs. an actual problem with my code.  I'm working on fixing this and the provide.

Reply:
If you'd like to add a commit that updates `tools/sanity_checks.py` to wrap that json.load in a try/except block and have it raise a slightly more informative error -- say, for example, "file 'releases.json' is malformed", then people would not make that mistake in the future, and future you might thank present you the next time you submit a wrap update. :D